### DISCUSSION TITLE
Will MFA be added to Poweradmin

### DISCUSSION THREAD

**Discussion by faniedp741030** [poweradmin/poweradmin#597] [Ideas]
Good day,
We have a requirement for MFA on Poweradmin due to security compliance.
Will this be added in future releases

**Comment by edmondas** [comment]
> Good day, We have a requirement for MFA on Poweradmin due to security compliance. Will this be added in future releases

I have several features planned, but their rollout depends on the codebase's readiness. While my next month is already scheduled with development tasks, I'd like to understand your basic compliance needs.

Would you help outline the essential specifications? For MFA, we should first establish the primary delivery method - whether it's SMS, email, or an authentication app. Once that's in place, we can address the account recovery workflow in a later phase.

Please let me know what core features are required for compliance and what's the deadline?

**Reply by faniedp741030** [reply]
Hi,

Some of clients have requested for MFA authentication and our holding company has a new policy that MFA is a requirement,email and or authentication app.
As for the deadline, if you can maybe provide a timeline that will be fine.
I do understand that there is a lot of features you are planning. I would just like to give our customers and holding company an idea when it would be implemented.

**Reply by edmondas** [reply]
I plan to begin implementation in early 2025, with Q1 dedicated to developing the initial version of Multi-Factor Authentication (MFA). However, I'll need your involvement, as users like you best understand the requirements and can provide valuable feedback to shape the solution.

**Reply by faniedp741030** [reply]
Hi Sorry I missed the last message currently email mfa is fine with me

**Comment by edmondas** [comment]
Added ticket #623 to next milestone to track requirements from discussion

**Comment by edmondas** [comment]
The initial solution has been merged to master. However, please note that all code requires proper testing and stabilization before it can be considered production-ready.
For detailed information on implementation specifics, current status, and remaining work, please refer to the related ticket #623
